### ISSUE TITLE
[Bug] Inline blame doesn't show up while coding on IntelliJ 2018.2

### ISSUE THREAD

**Issue by hernanBeiza** [zielu/GitToolBox#251]
Hi! First of all, this plugin is awesome 🎉. Works really great for branch pulling and pushing. Thanks for your work!
However, a few months ago the inline blame feature doesn't show up while coding on IntelliJ 2018.2, running on OSX, Ubuntu 20.04 and Windows 10.
I tried reinstalling it but doesnt worked.

I hope you can help me. 
Thanks for your time!

**Comment by zielu** [comment]
Hi, I'll take a look over the weekend.

**Comment by whisperice** [comment]
The same. Inline blame for Pycharm 2020.1.3, OSX 10.15.4 not working from today. It worked yesterday, but when I restart mac today, the inline blame has gone. 
BTW, I have IntelliJ and Webstorm in the same mac, inline blame still work for these two.

**Comment by whisperice** [comment]
Ah, sorry man. I just notice there is a checkbox for the inline blame.
I didn't change the setting but it unchecked. Now it works after I check it. Awesone. @hernanBeiza did you have a look at this setting?
I didn't make any change to the plugin setting when I installed the plugin and the inline blame just work. I think it was checked by default but not sure why it is unchecked now. 
![image](https://user-images.githubusercontent.com/20794360/87114552-7664d980-c2a3-11ea-9987-dc879a1ed308.png)

**Comment by hernanBeiza** [comment]
I don't have that option on my settings 🤕
![image](https://user-images.githubusercontent.com/1325005/87156008-f41beb80-c289-11ea-9cf5-e318a77b329a.png)

**Comment by hernanBeiza** [comment]
> Ah, sorry man. I just notice there is a checkbox for the inline blame.
> I didn't change the setting but it unchecked. Now it works after I check it. Awesone. @hernanBeiza did you have a look at this setting?
> I didn't make any change to the plugin setting when I installed the plugin and the inline blame just work. I think it was checked by default but not sure why it was unchecked today.
> ![image](https://user-images.githubusercontent.com/20794360/87114552-7664d980-c2a3-11ea-9987-dc879a1ed308.png)

@whisperice I don't have that option in my settings 🤕
![image](https://user-images.githubusercontent.com/1325005/87156205-4fe67480-c28a-11ea-976b-851a293eaf3c.png)

**Comment by whisperice** [comment]
> > Ah, sorry man. I just notice there is a checkbox for the inline blame.
> > I didn't change the setting but it unchecked. Now it works after I check it. Awesone. @hernanBeiza did you have a look at this setting?
> > I didn't make any change to the plugin setting when I installed the plugin and the inline blame just work. I think it was checked by default but not sure why it was unchecked today.
> > ![image](https://user-images.githubusercontent.com/20794360/87114552-7664d980-c2a3-11ea-9987-dc879a1ed308.png)
> 
> @whisperice I don't have that option in my settings 🤕
> ![image](https://user-images.githubusercontent.com/1325005/87156205-4fe67480-c28a-11ea-976b-851a293eaf3c.png)

seems like you are using an old verison of plugin?

**Comment by hernanBeiza** [comment]
I am using 182.7.1 version over IntelliJ Ultimate 2018.2.8
![image](https://user-images.githubusercontent.com/1325005/87308755-b2897b80-c4e9-11ea-9078-325b1388a51e.png)

**Comment by zielu** [comment]
Hi, I don't release new versions for that one. It's policy to reduce maintanence and don't deal with API changes.

**Comment by hernanBeiza** [comment]
Ops... I am sorry. I didn't know that! Thanks for your kindness and time!

**Comment by zielu** [comment]
I did some digging and inline blame was added in 2018.3 :)

**Comment by ryenus** [comment]
**TIL**: for multi-module projects, I had to enable the top module to get inline blame to work.

Before that I only had one of the submodule enabled and the inline blame didn't work, which took me sometime to figure out what went wrong, and that's why I found this issue. Anyway, turned out enabling the top module solved the issue.

**Comment by ryenus** [comment]
**TIL**: for multi-module projects, I had to enable the top module to get inline blame to work.

Before that I only had one of the submodule enabled and the inline blame didn't work, which took me sometime to figure out what went wrong, and that's why I found this issue. Anyway, turned out also enabling the top module solved the issue.

**Comment by zielu** [comment]
@ryenus which settings did you have to change ? Maybe I could improve it

**Comment by ryenus** [comment]
@zielu no, I did not change any settings. 

What I did was, in the **Load/Unload Modules** dialog, I moved the top module from the **Unloaded modules** to the **Loaded modules**:

![image](https://user-images.githubusercontent.com/610161/99471996-ae330a80-2982-11eb-9450-a60057fcabd6.png)

**Comment by hernanBeiza** [comment]
> I did some digging and inline blame was added in 2018.3 :)

I will check this soon and I write you later. Thanks for your info

**Comment by hernanBeiza** [comment]
> I did some digging and inline blame was added in 2018.3 :)

@zielu I checked 2018.3 and works like a charm! 

![Captura de pantalla 2020-11-23 a la(s) 11 32 01](https://user-images.githubusercontent.com/1325005/99974510-cb4f4b00-2d7f-11eb-9e8d-c11c4d8e8678.png)

Anyway I unchecked "Show 'Git Stat' in  status bar". Thanks for your info @whisperice

![Captura de pantalla 2020-11-23 a la(s) 11 32 37](https://user-images.githubusercontent.com/1325005/99974604-f174eb00-2d7f-11eb-93d8-483cb0f684de.png)

**Comment by jaggaer-npasic** [comment]
![image](https://user-images.githubusercontent.com/70564484/100452898-54d18680-30ba-11eb-8e7a-2beec1c233b5.png)

I have everything configured but it's not working....
Version 202.2.4

**Comment by zielu** [comment]
@jaggaer-npasic does it show up in status bar or is it missing altogether ?

**Comment by jaggaer-npasic** [comment]
@zielu 
It's visible
![image](https://user-images.githubusercontent.com/70564484/100460503-91f04580-30c7-11eb-888a-8a75809a3246.png)

**Comment by zielu** [comment]
Hmm, I don't see blame component in there. I noticed you are using Material UI - could you try to change to other theme ? There were problems with it in the past.

If it does not help could you enable debug logging and let's see if there is something interesting in there: https://github.com/zielu/GitToolBox#debug-logging

**Comment by jaggaer-npasic** [comment]
Hi, 
I've used different themes and also, default one, but the result is the same.
The only thing I've noticed is that if you import multiple projects only the first one will have this feature and the others don't.
The first one is the only one that could be imported as "Project", the others are imported as modules.
If you try to create some dummy project, in which you will import your projects as modules, Again, only the one that is imported first will have this feature working.
I've used Intellij Ultimate 2019, IntelliJ Community edition (many 2020 versions), and also, now Ultimate 2020. In IntelliJ Ultimate 2019 everything was working perfectly, At that time, I've shown your plugin to my colleagues, and all of them are using this plugin since then.
